index.html

```diff
@@ -3,8 +3,38 @@
     <head>
         <meta name="viewport" content="width=device-width, initial-scale=1" charset="utf-8">
         <link rel="stylesheet" href="stetics.css"> 
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1>Hello Microverse!</h1>
+        <h1 class="header">
+            <p class="logo">My Logo</p>
+            <div><img class="menu" src="Union.png" alt="MenuBar"></div>
+        </h1>
+        <div class="notheader">
+            <div class="textcontainer">
+                <div class="primarytext">Hey There.<br>
+                    I'm Gabriel.
+                </div>
+                <div class="supportingtext">
+                    I'm a Software Developer
+                </div>
+                <div class="description">
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don't hesitate to contact me.
+                </div>
+            </div>
+            <ul class="socialmedia">
+                <li><img class="mediaicon" src="Disabled.png" alt="SociaMediaIcon"></li>
+                <li><img class="mediaicon" src="Disabled-1.png" alt="SociaMediaIcon"></li>            
+                <li><img class="mediaicon" src="Enabled.png" alt="SociaMediaIcon"></li>
+                <li><img class="mediaicon" src="Disabled-2.png" alt="SociaMediaIcon"></li>
+                <li><img class="mediaicon" src="Disabled-3.png" alt="SociaMediaIcon"></li>
+            </ul>
```

Comment:
Taking a closer look at the headline section, I feel there are couple of things that requires your attention for them to look better :

  - The illustration and the text are currently overlapping, which doesn't provide a good UX because a visitor wants to read about you but the illustration is obstructing and making the text difficult to read. Kindly address this to improve the UX.
  - The illustration is not positioned correctly as seen in the figma design and I think part of the issue is because you haven't set the `div` with the `notheader` class height to `100vh`. Kindly do so to make your design match the given template.
  - Kindly improve the gap between the I am a developer text and the paragraph in order to make improve the text heirarchy.

YOU DESIGN
![eweeeh](https://user-images.githubusercontent.com/84930355/227289582-c7226af8-e186-4c28-8d21-a726927ab9ef.PNG)

FIGMA DESIGN
![save](https://user-images.githubusercontent.com/84930355/227290129-9ab6a3ae-b2e6-4dcc-9f44-e40698a7eb63.PNG)